src/Resources/Parcel.php

```diff
@@ -85,6 +85,19 @@ public function ageCheck(): self
         return $this;
     }
 
+    /**
+     * Set Insurance for parcel (only applicable for package_type 1)
+     * @param int $amount - in cents
+     * @param string $currency
+     * @return $this
+     */
+    public function insurance(int $amount, string $currency = 'EUR'): self
```

Comment:
Maybe it is better to use `$cents` here instead of `$amount`, this indicates better that we expect a value in cents here.